website/project/model.py

```diff
@@ -615,6 +655,9 @@ class Node(GuidStoredObject, AddonModelMixin):
     # {<User.id>: [<Node._id>, <Node2._id>, ...] }
     child_node_subscriptions = fields.DictionaryField(default=dict)
 
+    # Files that contains comments
+    commented_files = fields.DictionaryField(default=dict)
```

Comment:
Is there a way to get comment counts for a node's files without having to maintain this additional `DictionaryField`?